pkgs/by-name/mc/mcap/package.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  zlib,
+  zstd,
+  lz4,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "mcap";
+  version = "0.0.53";
+
+  src = fetchFromGitHub {
+    owner = "foxglove";
+    repo = "mcap";
+    rev = "releases/mcap-cli/v${finalAttrs.version}";
+    hash = "sha256-1NJ50dgbizOSl0ZMoqmuQxHc7Ca5JgS1bJ7tREDaKIU=";
+    fetchSubmodules = true;
+  };
+
+  sourceRoot = "source/cpp";
+
+  # Skip the build phase entirely
+  dontBuild = true;
+
+  # Skip the configure phase
+  dontConfigure = true;
```

This feels like the wrong order

```suggestion
  # Skip the configure phase
  dontConfigure = true;

  # Skip the build phase entirely
  dontBuild = true;
```